public/index.css

```diff
@@ -0,0 +1,4 @@
+.test
```

Comment:
is this used?

Reply:
Blame Ralph

Reply:
If it's not used, we'd better remove it from this pull request.

Reply:
Agreed